src/assets/images/home/index.js

```diff
@@ -18,8 +19,14 @@ const LaptopImg = styled.img.attrs({
   src: LaptopPNG,
 })``;
 
+const PhoneImg = styled.img.attrs({
+  alt: 'laptop',
```

Comment:
shouldn't this alt be `phone` or something?

Reply:
oops yes, thank you!